### PR TITLE
Add "Add to clipboard" button on code snippets, improve test-results page

### PR DESCRIPTION
**Pull request by elessar-ch**
screen shot 2016-11-15 at 10 20 36

### CODE REVIEW

File: tests/template.hbs [hunk 1]
```diff
@@ -57,8 +57,10 @@
             </div>
           {{else}}
             <p class="passed">
-              Test passed with flying colors.
+              Test passed with flying colors. There was no differeence detected
```
Comment (FKSI) [inline tests/template.hbs]: Small typo there no big deal 👍

Comment (elessar-ch) [reply tests/template.hbs]: thx @FKSI

**Comment by davidknezic** [comment]
Looking into it...

**Comment by davidknezic** [comment]
The "Copy to clipboard" button is big! I made it more subtle.

## Before

screen shot 2016-11-15 at 09 51 28

## After

screen shot 2016-11-15 at 10 20 36

**Review by davidknezic — APPROVED** [review]
ok now

**Pull request merged**